Berry Nice Metastation Crew Quarters Overhaul

BotTierPC opened the pull request:
## About The Pull Request

Where to BEGIN! I had originally taken issue with the odd placement of the laundry machine, the scattered tables and EMPTY feel of the locker room, and i wanted **better**. Maybe not better, but more, so I started with an overhaul of the washroom and extended out from there, encompassing the entirety <sub>(-pool area/holodeck)</sub> of the Living Quarters and not only giving it a fresh coat of paint but helping return it to its origins of a living space. 

YES! We are on a WORK STATION, but the Crew Quaters should be a place where people **not** on shift would be, while not making it so comfortable that people are ditching work to go to however. So with that theme in mind this PR renovates the space. 

## Why It's Good For The Game

Not only does it look **NICER** <sub>personally speaking</sub> but honestly the old layout was WACK. Random spare machines stored, haphazard table layout and a discount botony room attached. Covered up with a multitude of uninspired dorm rooms. MetaStation Service has <sub> ( and is at time of writing ) </sub> being re-worked, and it is beautiful, so much of modern mapping is, from Cardinal Station, the aforementioned rework, Radstation's recent*ish* big change, all of these bring the standard and bar for good looking rooms, and sections of station UP. Its only fair that more locations are opened up to the same treatment and modernization.

## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>
<summary>Screenshots&Videos</summary>

### Entrance / Locker Room

<img width="450" height="580" alt="Screenshot 2025-09-25 170326" src="https://github.com/user-attachments/assets/a719e1dc-225a-4c42-af8e-f3311b7d1850" />
<img width="1107" height="657" alt="Screenshot 2025-09-25 170344" src="https://github.com/user-attachments/assets/82fcf146-eefb-4a54-83e6-67c3484ce959" />


### Washroom

<img width="519" height="459" alt="Screenshot 2025-09-25 155939" src="https://github.com/user-attachments/assets/32360738-4d3c-4b0f-aa54-2566564787f3" />
<img width="343" height="351" alt="Screenshot 2025-09-25 160005" src="https://github.com/user-attachments/assets/a6a3efa7-f5af-4546-b758-3e606d2ca108" />
<img width="351" height="342" alt="Screenshot 2025-09-25 160021" src="https://github.com/user-attachments/assets/e5eb4a54-d591-49a5-b1e7-aed26be2722f" />

### Garden

<img width="907" height="679" alt="Screenshot 2025-09-25 160957" src="https://github.com/user-attachments/assets/f7f2debf-0fbd-4c5e-b040-f73b7bd5d366" />

### Dorms

<img width="840" height="839" alt="Screenshot 2025-09-25 161146" src="https://github.com/user-attachments/assets/a512c57b-5d98-4536-b53d-4077587f00f0" />
<img width="341" height="460" alt="Screenshot 2025-09-25 161204" src="https://github.com/user-attachments/assets/1c8f8067-a2cf-4e21-8e54-013dcd452fbe" />
<img width="867" height="733" alt="Screenshot 2025-09-25 161255" src="https://github.com/user-attachments/assets/8226d38a-8354-4b24-8a65-cd752d190b86" />

### Strong.dmm

<img width="622" height="566" alt="Screenshot 2025-09-25 161406" src="https://github.com/user-attachments/assets/11f73621-6302-46a2-8381-3552593e01d8" />

</details>

## Changelog
:cl:
tweak: Overhauls Meta-Station Dorms, Washrooms, Garden, and Locker Room.
/:cl:

BotTierPC committed:
Re-Work of Metastation Crew Quaters

Huge overhaul of meta stations dorm, washrooms, garden and locker room.

BotTierPC committed:
Fixes some decals in locker room

BotTierPC committed:
Fixes some issues + 1 change

APCS are now the auto detecting kind, disposals is no longer wired, garden gets wood floor, wire in pool room fixed, air alarm in dorms moved off of unreachable wall

BotTierPC committed:
TRYS AGAIN TO DELETE THAT TABLE

also fixes the orientation path or something of a hydroponics wall closet

BotTierPC committed:
DELETES THE MOTHING TABLE

Fixes a pipe, sets the tv to NULL rather then a blank list of channels, AND TRYS TO DELETE THE TABLE CAUSING ERRORS

BotTierPC committed:
Puts MothNews6 on the TV

TV is causing an error due to me trying to have it not assigned to any channels. i tried blank, then setting the vv to null - both caused errors. So trying a network with no cameras connected (aka nonsense network)

BotTierPC committed:
Ends MothNews6 for a real segment found in code called LIMBO

Rather then just try and gaslight code, i looked at the books, found the pre-existing channels, and am putting it up against the gitbot with nothing but prayers

BotTierPC committed:
Tries ONE LAST TIME (i promise) to change the network

set to that BOT channel, idk maybe the idea of watching borgs is funny

BotTierPC committed:
....returns the tv to ss13 network

so yeah, its just a camera console at this point BUT IT LOOKS NICE - and, to justify my late night rational for just wanting to see the bot say good job, there is none, thats all i want at this point ill think about the rest next work cycle

BotTierPC committed:
Swaps Machinery locations

The Garden Seed Extrctor is put back in the work cubby, and Biogen is moved closer to the door and plants for ease of access

BotTierPC committed:
Minor Tweaks / Fixes

Rukofamicom reviewed (changes requested):
Hard block on the public garden getting any of the following botany tools:

* ~~Trays instead of dirt~~ Botany trays that weren't already there
* Botany chem dispenser
* Machinery dedicated to modifying plants in any way.

BotTierPC committed:
aforementioned changes

BotTierPC committed:
Fixes Network error and adds tile replacement

details to be found within comment.

BotTierPC committed:
Common Room Error Fix / Wall Rearrangement

Moves a Camera one tile left now above the pet bed. Adds a light fixture above disposals, and removes the small light fixture previously attached to a glass wall between the dorm doors.

BotTierPC committed:
Merge remote-tracking branch 'upstream/master' into BerryKiloGammaIV

BotTierPC committed:
updates something

i honestly do not know what but mrman said to

BotTierPC committed:
Fixes the incoming smite from crossedfall

MOVES A SCRIPT INTO THE SCRIPTS FOLDER

BotTierPC committed:
Changes Matching Current Photos

brings changes and fixes made up to date with current evidence provided

BotTierPC committed:
Fixes Error?

Deletes error causing pixel_ code attached to a poster and urinals

BotTierPC committed:
Fixes Plant Being Zoned As Space

BotTierPC committed:
Final Mini-Changes!

matches the shower decals to those around urinals - de-aligns the trash in locker room - removes floor lights in toliet stalls and adds wall lights

BotTierPC committed:
Raspberry Washes Their Hands With This

i SWEAR this is the final bit of changes i push. The next addition is either fixing an error, addressing another persons commentary, or changes after a tm or whatevs I VOW IT

BotTierPC committed:
Tiny Fix / Change

Swaps the washroom decorations - makes the toliet bolts / button work

BotTierPC committed:
Merge remote-tracking branch 'upstream/master' into BerryKiloGammaIV

BotTierPC committed:
Pixel Bug Fix?
